@app/server/package.json

```diff
@@ -5,10 +5,10 @@
   "scripts": {
     "build": "tsc -b",
     "start": "node -r @app/config/env dist/index.js",
-    "dev": "nodemon --signal SIGINT --watch 'dist/**/*.js' -x 'node --inspect=9678 -r @app/config/env -r source-map-support/register' dist/index.js",
-    "schema:export": "node -r @app/config/env node_modules/.bin/ts-node -O '{\"rootDir\":null}' scripts/schema-export.ts",
+    "dev": "nodemon --signal SIGINT --watch 'dist/**/*.js' -x \"node --inspect=9678 -r @app/config/env -r source-map-support/register\" dist/index.js",
+    "schema:export": "npx --no-install -n \"-r @app/config/env\" ts-node -O '{\"rootDir\":null}' scripts/schema-export.ts",
```

Comment:
I notice in the 'dev' line you've replaced `'` with `\"`; but the `-O` argument doesn't have the same change for `schema:export`. Is replacing `'` required? If so: why is it not required here, or was it simply missed? If not: we should go back to `'` because it's simpler.

Reply:
Indeed it was not working on Windows as there everything needs to be double quotes in the package.json. So with all the escaping we need to use 6 backslashes unfortunately. It was the only way I could make it work